ui/src/app/richskill/import/BatchImportOptionsEnum.ts

```diff
@@ -0,0 +1,5 @@
+export enum BatchImportOptionsEnum {
```

Comment:
one of these options should be for `rsd library`

Reply:
I think it is not needed since that is the default option in the Batch import screen.